tests/Tests/Client/KeenIOClientAccessKeysTest.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+namespace KeenIO\Tests\Client;
+
+use KeenIO\Client\KeenIOClient;
+use GuzzleHttp\Subscriber\Mock;
+use GuzzleHttp\Handler\MockHandler;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Psr7\Response;
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Command\Guzzle\GuzzleClient;
+
+class KeenIOClientAccessKeysTest extends \PHPUnit_Framework_TestCase
```

Comment:
To be compatible with the latest PHPUnit version, I think we have to use the `PHPUnit\Framework\TestCase` instead.

Reply:
i think it is impossible to do, because compose.json has a dependency on `phpunit/phpunit": "5.3.*`